src/logging.c

```diff
@@ -518,6 +529,24 @@ void rcutils_logging_console_output_handler(
       token_expansion = name;
     } else if (strcmp("message", token) == 0) {
       token_expansion = message_buffer;
+    } else if (strcmp("seconds", token) == 0) {
+      if (snprintf(
+          numeric_storage, sizeof(numeric_storage),
+          "%f", timestamp / 1e9) <= 0)
+      {
+        RCUTILS_SAFE_FWRITE_TO_STDERR("Failed to convert float to string.\n");
+        goto cleanup;
+      }
+      token_expansion = numeric_storage;
+    } else if (strcmp("nanoseconds", token) == 0) {
```

Comment:
I guess users might use `{seconds}.{nanoseconds}` but then get confused by the variable nature of digits of the nanoseconds part. I think an additional placeholder is desired / necessary which pads the output with leasing zeros to a fixed length of 9 digits.

Reply:
I think you're confusing the purpose of `{nanoseconds}`, it's the total timestamp in nanoseconds, not the nanoseconds part of time. `{seconds}` is floating point time. So, `{seconds}.{nanoseconds}` doesn't really make any sense. See the documentation:

https://github.com/ros2/rcutils/pull/85/files#diff-f1d39d4e841937fde043a8e49b7e6c2aR62

Reply:
Oh, got it now. Thanks for clarifying.

Reply:
that was how I also first interpreted it (as sec.nanosec), so if it's happened to a few people maybe there's a way we can rename it to avoid confusion. We could just do `time` like in ROS 1 and additionally `time_ns`? Then you'd be less likely to think `{time}.{time_ns}`, but 'time' is not as descriptive as we'd like moving forward..

Reply:
It's a fair point, however, that the padding is never considered. So on the floating point on the seconds may not be consistent from machine to machine. I'll see what I can do about that, as ideally it would be similar.

I'm actually writing the missing test right now, I so I'll think if a fixed padding makes sense in either case.

Reply:
It doesn't need to be concise I don't think, we could use `timestamp` and `timestamp_as_nanoseconds`.

Reply:
@dhood and I talked about it offline and we decided on `time` and `time_as_nanoseconds`, using `time` rather than `timestamp` to be closer to what ROS 1 does, and using `_as_nanoseconds` to be maximumly specific without worrying about verbosity too much because we expect it to not be used that much in favor of the floating point seconds version.